org-static-blog.el

```diff
@@ -438,7 +442,7 @@ Preamble and Postamble are excluded, too."
 For example, when `org-static-blog-publish-url` is set to 'https://example.com/'
 and `relative-url` is passed as 'archive.html' then the function
 will return 'https://example.com/archive.html'."
-  (concat org-static-blog-publish-url relative-url))
+  (concat-to-dir org-static-blog-publish-url relative-url))
```

Comment:
`org-static-blog-publish-url` is not a directory. Does it still make sense to use `concat-to-dir`?

Reply:
Yes, `org-static-blog-publish-url` is not a directory but it has the properties
of a directory. No matter how a URL is written it should be interpreted with a
forwardslash at the end